web/app/dao/DatasetsDAO.java

```diff
@@ -55,11 +55,11 @@
 			"GROUP_CONCAT(IFNULL(u.display_name, '*') ORDER BY o.sort_id ASC SEPARATOR ',') as owner_name, " +
 			"FROM_UNIXTIME(source_created_time) as created, d.source_modified_time, " +
 			"FROM_UNIXTIME(source_modified_time) as modified " +
-			"FROM dict_dataset d " +
+			"FROM ( SELECT * FROM dict_dataset ORDER BY urn LIMIT ?, ? ) d " +
```

Comment:
This is sufficient for now.

We will add more API to perform lookups for other attributes. Then we can switch to simpler pagenation SQL in the future.